CruCentralCoast/Events/EventsVC.swift

```diff
@@ -7,12 +7,20 @@
 //
 
 import UIKit
+import RealmSwift
 
 class EventsTVC: UITableViewController {
     var selectedCell: EventsTableCell?
     var originFrame: CGRect?
     
-    var dataArray = [Event]()
+    var dataArray: Results<Event>!
+    var people: Results<Person>!
```

Comment:
Why is all this stuff in EventsVC? Is it because this is the first tab that is loaded? Or will this code be necessary in all the tabs?

Reply:
I just read the commits log. Was this put in EventsVC for testing?

Reply:
Yes, I temporarily added this while I test some final stuff. EventsVC is the first UIViewController in the app, so it was easiest to test everything right there